bigquery/query.rb

```diff
@@ -31,22 +31,28 @@ def unique_words project_id
         # [END build_service]
 
         # [START run_query]
-        sql = "SELECT TOP(corpus, 10) as title, COUNT(*) as unique_words " +
-              "FROM [publicdata:samples.shakespeare]"
         results = bigquery.query sql
         # [END run_query]
 
         # [START print_results]
-        results.each do |row|
-          puts "#{row['title']}: #{row['unique_words']}"
+        results.each_with_index do |row, index|
```

Comment:
The index adds unnecessary complexity to listing results.  Any reason why the index is needed?  I'd love to simply see `results.each do |row|`